test/implementation-tests.js

```diff
@@ -847,6 +847,17 @@ describe("Tests that include infinite recursion", () => {
     });
 });
 
+describe("Tests converting an empty array", () => {
+    // When converting an empty array of API response to an object with JSONata
+    // the library should return an empty object instead of throwing an error
+    // https://github.com/jsonata-js/jsonata/issues/547.
+    it('should return empty object', () => {
+        var expr = jsonata("$${id:{'label':label,'value':value}}");
+        var result = expr.evaluate([]);
+        expect(result).to.deep.equal({});
+    });
+});
+
```

Comment:
Many thanks for adding the test case.  But since it's not an implementation specific test, please could you add it as an implementation agnostic test case in JSON format.  I suggest adding a new file `jsonata/test/test-suite/groups/object-constructor/case026.json` containing the following:
```
{
    "expr": "$${id:{'label':label,'value':value}}",
    "data": [],
    "bindings": {},
    "result": {}
}
```

Reply:
Thanks @andrew-coleman I updated the PR to add this new file and removed the test case from implementation-tests.js.